python/lsst/afw/geom/ellipses/ellipsesLib.py

```diff
@@ -1 +1,5 @@
-from __future__ import absolute_import
\ No newline at end of file
+from __future__ import absolute_import
+
+from ._baseCore import *
+from ._quadrupole import *
+from ._ellipse import *
```

Comment:
It avoids some clutter in the `__init__.py` file, so it's probably reasonable (though that clutter has to go somewhere, so it's not a big win). It's fine with me to leave it, but a "to do" item is to update of our coding standards once we figure out what we want. There are other instances that are more of an issue; @fred3m fixed them in a nice way for this ticket but @pschella has said that technique may not work for other pybind11 code.